docs/guide/docs/introduction.md

```diff
@@ -3,35 +3,40 @@ id: introduction
 title: Introduction
 ---
 
-Building bots is something new and sounds like a fun challenge for most developers. To be able to create software that converses in a natural way with humans is an exciting new reality. The problem is that you don’t want to spend hundreds of hours learning and putting together all the infrastructure and boilerplate code required to get a basic conversational backend up and running.
+Building chatbots is a new, exciting challenge for most developers. Creating software that holds a conversation and responds naturally to humans is a thrilling possibility, and Botpress is turning this into reality. 
+
+![Simple Pizza Bot](assets/pizzabot.gif)
+
+The catch is that you (or anyone else) wouldn't want to spend hundreds of hours learning and putting together all the infrastructure and boilerplate code required to get a conversational backend up and running.
 
 ## What is Botpress
 
-> Botpress is a **lightweight**, **fast** and **flexible** on-premise bot building platform.
+> A **lightweight**, **fast**, and **flexible** chatbot building framework that you can host anywhere, whether it be on the cloud or your servers.
 
-It is a complete platform that ships with all the tools you need to build, deploy and manage production-grade bots in record time. Here are some of the components that are installed by default:
+It is a complete platform that ships with all the tools you need to build, deploy, and manage production-grade bots in record time. Here are some of the components that are installed by default:
 
-- An NLU Engine
-- An administration dashboard
-- A visual flow editor
-- A chat emulator / debugger
-- Support for multiple messaging channels
+- An NLU Engine which understands the meaning and intentions behind user statements even where grammatical mistakes exist.
+- An administration interface that can be viewed from a browser, exposing modules to manage content and show chatbot  statistics, among other things. 
+- A visual workflow builder used for creating conversations and other interactions.
+- A chat emulator/debugger to simulate conversations and inspect the chatbot engine's logic.
+- Support for popular messaging channels like Facebook Messenger, Slack, Telegram, and others.
```

Comment:
I know that's picky but can you add Microsoft Teams here as well as this is quite an important channel for enterprise users